code/game/objects/buckling.dm

```diff
@@ -40,11 +40,23 @@
 	if(!istype(M))
 		return FALSE
 
+	if((buckle_restrictions & CANT_BUCKLE_SELF) && M == usr && !force)
+		to_chat(usr, "<span class='warning'>You are unable to buckle yourself to [src]!</span>")
```

Comment:
this is such a hack, pass mob doing the buckling in to the proc as well as the buckle target.

usr is a pos hack

Reply:
@armhulen if you want I can PR a fix

Reply:
also, whats a pos hack

Reply:
piece of shit